core-java-modules/core-java-lang-3/src/main/java/com/baeldung/transientkw/Book.java

```diff
@@ -11,6 +11,8 @@ public class Book implements Serializable {
     private transient int copies;
     private final transient String bookCategory = "Fiction";
 
+    private final transient String bookCategoryNewOperator = new String("Fiction with new Operator");
```

Comment:
I'm not sure that is correct.
What is the purpose for the `final` immutable object use transient?

Reply:
Hi @andrzejsydor, 

Thank you for your feedback!
You are right that Strings are by default immutable. 
The purpose of the `transient` is to not include that field in the serialization/de serialialization proces.
Could you please tell me what is your thoughts about incorrect part?

Reply:
true, that's right, sorry my mistake